ui/partials/main/thActionButton.html

```diff
@@ -10,10 +10,12 @@
   </button>
   <!-- Menu contents -->
   <ul class="dropdown-menu pull-right">
-    <li><a target="_blank" data-ignore-job-clear-on-click
+    <li title="{{user.loggedIn ? '' : 'Must be logged in'}}"
```

Comment:
Title needs to get applied here because disabling pointer events on the <a> tag makes tooltips not show, among other things.